tests/unit/test_thread_pools.cpp

```diff
@@ -30,7 +29,6 @@ class ThreadPoolsTest: public ::testing::Test {
 	static void SetUpTestCase() {
 		Alloc_Reset();
 		Config_Init(NULL, NULL, 0);
```

Comment:
Will this test pass if we remove the call to `Config_Init(NULL, NULL, 0);` ?

Reply:
I still need to check this